Gemfile.lock

```diff
@@ -100,7 +99,7 @@ GEM
       rubocop (~> 0.64)
       rubocop-rspec (~> 1.28)
       scss_lint
-    govuk_app_config (2.0.2)
+    govuk_app_config (2.0.3)
```

Comment:
This slipped in probably because someone forgot to run `bundle install` before their change got merged to master.